.github/workflows/cfn-nag.yml

```diff
@@ -0,0 +1,54 @@
+name: CFN Nag
+
+on:
+  push:
+    paths:
+    - "test_infra/**"
+  workflow_dispatch:
```

Comment:
Does this line achieve anything given there is nothing nested?

Reply:
It allows us to trigger the `cfn_nag` job manually from Github Actions (https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch). I can remove if this is unnecessary.

Reply:
Ah yeah fair enough, it would indeed be good to run it manually too